com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -272,6 +272,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed remapping of depth pyramid debug view
 - Fixed an issue with asymmetric projection matrices and fog / pathtracing. (case 1330290).
 - Fixed gbuffer depth debug mode for materials not rendered during the prepass.
+- Fixed NVIDIA install button, changed it to a Fix button and also prevent user from spamming and corrupting installation of nvidia package.
```

Comment:
If I were you I would split the changelog in two.
```
### Fixed
- Prevent user from spamming and corrupting installation of nvidia package.

### Changed
- Changed the NVIDIA install button to the standard FixMeButton.
```
As the change of the button is not a fix

Reply:
Done